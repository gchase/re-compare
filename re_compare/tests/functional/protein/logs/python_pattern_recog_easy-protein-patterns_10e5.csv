regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,3.5970211029052734,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,3.5970211029052734,"[-1, -1]"
DHKNLD.,first_match,0,0.10919570922851562,"[-1, -1]"
DHKNLD.,all_matches,0,0.10919570922851562,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,3.113985061645508,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,3.113985061645508,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,0.011920928955078125,"[99268, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,3.125905990600586,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,0.2372264862060547,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,0.2372264862060547,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.0040531158447265625,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.0040531158447265625,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.0030994415283203125,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.007152557373046875,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,3.876924514770508,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,3.876924514770508,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,0.035762786865234375,"[50386, 50391]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,3.912687301635742,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.008344650268554688,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.008344650268554688,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.010967254638671875,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.019311904907226562,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,0.38909912109375,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,0.38909912109375,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,1.5978813171386719,"[58929, 58933]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,1.9869804382324219,"[-1, -1]"
P..L...KWE.,first_match,0,0.14781951904296875,"[-1, -1]"
P..L...KWE.,all_matches,0,0.14781951904296875,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,1.837015151977539,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,1.837015151977539,"[-1, -1]"
