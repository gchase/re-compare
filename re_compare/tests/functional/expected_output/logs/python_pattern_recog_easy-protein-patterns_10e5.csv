regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,4.235029220581055,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,4.235029220581055,"[-1, -1]"
DHKNLD.,first_match,0,0.1399517059326172,"[-1, -1]"
DHKNLD.,all_matches,0,0.1399517059326172,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,3.02886962890625,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,3.02886962890625,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,0.012636184692382812,"[99268, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,3.041505813598633,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,0.17833709716796875,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,0.17833709716796875,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.00286102294921875,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.00286102294921875,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.0016689300537109375,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.0045299530029296875,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,4.195213317871094,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,4.195213317871094,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,0.030040740966796875,"[50386, 50391]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,4.225254058837891,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.007867813110351562,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.007867813110351562,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.010013580322265625,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.017881393432617188,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,0.6999969482421875,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,0.6999969482421875,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,2.3190975189208984,"[58929, 58933]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,3.019094467163086,"[-1, -1]"
P..L...KWE.,first_match,0,0.1938343048095703,"[-1, -1]"
P..L...KWE.,all_matches,0,0.1938343048095703,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,1.8470287322998047,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,1.8470287322998047,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),first_match,1,0.006198883056640625,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,1,0.006198883056640625,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,2,0.0026226043701171875,"[996, 1004]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),all_matches,2,0.008821487426757812,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),first_match,1,1.8138885498046875,"[41351, 41357]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),consecutive,1,1.8138885498046875,"[41351, 41357]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),all_matches,1,3.9510726928710938,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),first_match,0,0.13971328735351562,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),all_matches,0,0.13971328735351562,"[-1, -1]"
A|G....G,first_match,1,0.0040531158447265625,"[15, 15]"
A|G....G,consecutive,1,0.0040531158447265625,"[15, 15]"
A|G....G,consecutive,2,0.00286102294921875,"[19, 19]"
A|G....G,all_matches,2,0.0069141387939453125,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,first_match,1,9.31406021118164,"[57782, 57786]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,1,9.31406021118164,"[57782, 57786]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,2,0.026941299438476562,"[58001, 58005]"
((CP|.)(S|T)N)(I|L|V)G,all_matches,2,9.341001510620117,"[-1, -1]"
